sbi2-2023-cytotable-poster/poster.html

```diff
@@ -0,0 +1,589 @@
+<p class="figcaption">
+Figure 2. Plot showing processing time duration for CytoTable and Pycytominer for various datasets.
+</p>
+<img src="https://raw.githubusercontent.com/d33bs/CytoTable-benchmarks/cytotable-scalability/notebooks/images/cytotable-and-pycytominer-comparisons-join-memory-size.png" width="85%" style="display: block; margin: auto;" />
+<p class="figcaption">
+Figure 3. Plot showing total memory consumption for CytoTable and Pycytominer for various datasets.
+</p>
+<p>CytoTable builds upon the shoulders of Pycytominer, helping to streamline the <code>SingleCells.merge_single_cells(...)</code> method.
+We decrease overall processsing completion time (Figure 2) and memory consumption (Figure 3) for large amounts of data by leveraging composable data stack elements.</p>
+</div>
+<div id="empowering-the-cytomining-ecosystem" class="section level1">
+<h1>Empowering the Cytomining Ecosystem</h1>
+<div id="orchestration-cytosnake" class="section level2">
+<h2>Orchestration: CytoSnake</h2>
+<p><strong><em>Authors:</em></strong> Erik Serrano, Jenna Tomkinson, Roshan Kern, Vince Rubinetti, Dave Bunten, Gregory P. Way</p>
+<img src="images/cytosnake_MP_processing_cropped.png" width="100%" style="display: block; margin: auto;" />
+<p class="figcaption">
+Figure 4. Diagram showing how CytoSnake orchestration applied in multiple pipelines.
+</p>
+<div class="container">
+<div class="text-block" style="padding-top:40px;">
+CytoSnake is an innovative tool for orchestrating high-dimensional cell morphology data processing pipelines, including those which leverage CytoTable and other applied usecases.
+</div>
+<div class="text-block" style="text-align:center;">
+<img src="images/qr_cytosnake.png" style="width:70%"/>
+</div>
+</div>
+</div>
+<div id="applied-research-nf1-schwann-cell-project" class="section level2">
+<h2>Applied research: NF1 Schwann cell project</h2>
+<p><strong><em>Authors:</em></strong> Jenna Tomkinson, Cameron Mattson, Erik Serrano,<br> Gregory P. Way</p>
+<img src="images/nf1_and_cfret_workflow.png" width="100%" style="display: block; margin: auto;" />
+<p class="figcaption">
+Figure 5. Diagram showing NF1 pipeline implementation details including CytoTable and Pycytominer.
+</p>
+<div class="container">
+<div class="text-block" style="padding-top:40px;">
+Comprehensive dataset analyses for cell painting assays, enabling the further understanding of cellular morphology NF1 Schwann cells and rare disease treatment.
+</div>
+<div class="text-block" style="text-align:center;">
+<img src="images/qr_nf1.png" style="width:70%"/>
+</div>
+</div>
+<p><br>
+<br>
+<br></p>
+</div>
+<div id="applied-research-pyroptosis-signature-project" class="section level2">
+<h2>Applied research: Pyroptosis signature project</h2>
+<p><strong><em>Authors:</em></strong> Michael J. Lippincott, Jenna Tomkinson, Gregory P. Way</p>
```

Comment:
```suggestion
<p><strong><em>Authors:</em></strong> Michael J. Lippincott, Jenna Tomkinson, Interstellar Collaborators*, Masafumi Tsuboi, Carla Basualto-Alarcon, Gregory P. Way</p>
```

There are many other authors for this project, but I wonder if this is sufficient to save space?